synapse/lib/view.py

```diff
@@ -23,9 +23,13 @@
         'name': {'type': 'string'},
         'parent': {'type': ['string', 'null'], 'pattern': s_config.re_iden},
         'creator': {'type': 'string', 'pattern': s_config.re_iden},
+        'layers': {
+            'type': 'array',
+            'items': {'type': 'string', 'pattern': s_config.re_iden}
```

Comment:
good catch. maybe ideally would be a different PR, but i'm guilty of this constantly so.. :D